src/osx/core/colour.cpp

```diff
@@ -13,127 +13,244 @@
 #include "wx/colour.h"
 
 #ifndef WX_PRECOMP
-    #include "wx/gdicmn.h"
+#include "wx/gdicmn.h"
 #endif
 
 #include "wx/osx/private.h"
 
+CGColorSpaceRef wxMacGetGenericRGBColorSpace();
+
+wxColorRefData::~wxColorRefData()
+{
+}
+
+class wxCGColorRefData : public wxColorRefData
```

Comment:
This one should have "u" in its name too.

Reply:
with CGColor and NSColor I though I'd stick with the native names, but for wxColourRefData I absolutely agree